modules/k3s/src/main/java/org/testcontainers/k3s/K3sContainer.java

```diff
@@ -0,0 +1,65 @@
+package org.testcontainers.k3s;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import lombok.SneakyThrows;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class K3sContainer extends GenericContainer<K3sContainer> {
+
+    private String kubeConfigYaml;
+
+    private static final String TOKEN = "fixedtoken";
+
+    public K3sContainer(DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DockerImageName.parse("rancher/k3s"));
+
+        addExposedPorts(6443, 8443);
+        setPrivilegedMode(true);
+
+        Map<String, String> tmpFsMapping = new HashMap<>();
+        tmpFsMapping.put("/run", "");
+        tmpFsMapping.put("/var/run", "");
+        setTmpFsMapping(tmpFsMapping);
+
+        setCommand(
+            "server",
+            "--no-deploy=traefik",
+            "--token=" + TOKEN,
```

Comment:
WDYT about either:
1. removing this flag and using `K3S_TOKEN`, so that our users can override it
2. not setting any token. IIRC k3s will auto generate it, and, since we obtain the config in `TOKEN`, we should get the value already
3. 
```suggestion
```

Reply:
You're right - I think I prefer 2. It seems to work, so I'm not sure why I bothered to set it in the first place 🙃